docs/Guides/creating_new_vendor_app.md

```diff
@@ -0,0 +1,137 @@
+# Creating New Vendor Application
+
```

Comment:
Creating a thread for @ncline-va 's comment
> The Guide itself seems to be laid out well to walk through the configuration, and auth url customizations. We haven't had a single Vendor that uses Java as far as I know, so Java and Spring is an interesting first demo.
PHP, .NET, and Go seem to be the primary languages used. Some Angular, React, or Vue also.
>
> I won't have a chance to fully test this out before I go on vacation, but I'm sure it's been tested well enough that it doesn't need me to do so. I've reviewed it for clarity - there's just a few points needing clarification or adjustment.

This sample app is based on the Telkom Print App. They requested Java. The target audience of the guide was users creating an app for their own customers. We happened to cover all the vendor SSO requirements and linked to the vendor guide multiple times so felt this guide might make sense here. 

The example app also has examples for calling the platform APIs as a partner service account and the currently logged-in user as well as using the marketplace APIs to check the user's access.

I missed submitting my comments last night when adding you to this PR. Do you think this belongs with the vendor docs that talk about creating product integrations or should it exist only with the platform docs?